2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,14 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let arr = [];
+
+    for(let i = 0; cities[i]; i++)
+    {
+        arr[i] = `The temperature in ${cities[i]} is ${temperatureService(cities[i])} degrees`
```

Comment:
Nice, you are writing your code neatly and as the exercise is requiring of you. An array of strings was asked to be returned and you are doing so. Well done